src/prefix-tree/node.ts

```diff
@@ -32,7 +31,7 @@ export function updateParent(node: Node, parent: Node): void {
 }
 
 export function removeDocument(node: Node, docID: string): boolean {
-  const index = node.docs.indexOf(docID);
+  const index = node.docs.findIndex(doc => doc === docID);
```

Comment:
Why did you change this?

Reply:
Ups, I used to use an object, then switched back to plain string and left that behind. Thanks for noticing!